hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip551/AtomicBatchTest.java

```diff
@@ -69,6 +72,42 @@
 
 @HapiTestLifecycle
 public class AtomicBatchTest {
+    @HapiTest
+    public Stream<DynamicTest> validateFeesForChildren() {
+        final double BASE_FEE_BATCH_TRANSACTION = 0.001;
+        final double BASE_FEE_HBAR_CRYPTO_TRANSFER = 0.0001;
+        final double BASE_FEE_SUBMIT_MESSAGE_CUSTOM_FEE = 0.05;
+
+        final var innerTxn1 = cryptoTransfer(tinyBarsFromTo("alice", "bob", ONE_HBAR))
+                .payingWith("alice")
+                .txnId("innerTxnId")
+                .blankMemo()
+                .batchKey("batchOperator");
+        final var innerTxn2 = submitMessageTo("topic")
+                .message("TEST")
+                .payingWith("bob")
+                .txnId("innerTxnId2")
+                .blankMemo()
+                .batchKey("batchOperator");
+        return hapiTest(
+                newKeyNamed("adminKey"),
+                newKeyNamed("submitKey"),
+                newKeyNamed("feeScheduleKey"),
+                cryptoCreate("batchOperator").balance(ONE_HBAR),
+                cryptoCreate("alice").balance(2 * ONE_HBAR),
+                cryptoCreate("bob").balance(4 * ONE_HBAR),
+                cryptoCreate("collector").balance(0L),
+                createTopic("topic")
+                        .adminKeyName("adminKey")
+                        .feeScheduleKeyName("feeScheduleKey")
+                        .withConsensusCustomFee(fixedConsensusHbarFee(ONE_HBAR, "collector")),
+                usableTxnIdNamed("innerTxnId").payerId("alice"),
+                usableTxnIdNamed("innerTxnId2").payerId("bob"),
+                atomicBatch(innerTxn1, innerTxn2).payingWith("batchOperator").via("batchTxn"),
+                validateChargedUsd("batchTxn", BASE_FEE_BATCH_TRANSACTION),
+                validateInnerTxnChargedUsd("innerTxnId", "batchTxn", BASE_FEE_HBAR_CRYPTO_TRANSFER, 5),
+                validateInnerTxnChargedUsd("innerTxnId2", "batchTxn", BASE_FEE_SUBMIT_MESSAGE_CUSTOM_FEE, 5));
```

Comment:
Do we not also want to check separate sigs here?

Reply:
You mean add more signatures and verify the price @mhess-swl ?

Reply:
Nevermind, I'm thinking of something that is probably outside the scope of this test. 👍